g3doc/user_guide/FAQ.md

```diff
@@ -84,6 +84,19 @@ Make sure that permissions is correct on the `runsc` binary.
 sudo chmod a+rx /usr/local/bin/runsc
 ```
 
+Before `release-20230522.0` gvisor did not use `-directfs=true` by default.
+Directfs enables the gvisor sandbox process to configure caps a bit
+differently. If you are using `-directfs=false` and your Kernel is configured
+with YAMA LSM (see https://www.kernel.org/doc/Documentation/security/Yama.txt,
+and https://man7.org/linux/man-pages/man2/ptrace.2.html) make sure
+`/proc/sys/kernel/yama/ptrace_scope` is at most in mode 1:
+
+```bash
+sudo cat /proc/sys/kernel/yama/ptrace_scope
+2
+sudo bash -c ‘echo 1 > /proc/sys/kernel/yama/ptrace_scope’
```

Comment:
This is using a quotation mark that isn't recognized by bash (`‘` rather than `'`). Please change it to `'`

Reply:
sorry for this one, fixed.